doc/man1/openssl-ciphers.pod.in

```diff
@@ -115,6 +115,8 @@ used. The format is described below.
 The cipher list consists of one or more I<cipher strings> separated by colons.
 Commas or spaces are also acceptable separators but colons are normally used.
 
+The cipher string may reference a cipher using its standard name.
```

Comment:
You should have a reference to *where* the standard names are listed, such as https://www.iana.org/assignments/tls-parameters/tls-parameters.xhtml#tls-parameters-4

Reply:
Good point! Updated docs